applicationinsights.dev.json

```diff
@@ -19,16 +19,38 @@
   "sampling": {
     "percentage": 100,
     "overrides": [
+      {
+        "telemetryType": "dependency",
+        "attributes": [
+          {
+            "key": "db.system",
+            "value": "postgresql",
+            "matchType": "strict"
+          }
+        ],
+        "percentage": 100
```

Comment:
for dev and pre-prod set dependencies to 100%, which means we don't need any overrides in place but makes it easier to see what is going on for prod